spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParser.scala

```diff
@@ -52,6 +53,7 @@ import scala.util.{Random, Try}
   *         NumericParser !
   *         StringParser !
   *         BooleanParser !
+ *        BinaryParser !
```

Comment:
```suggestion
  *       BinaryParser !
```

Reply:
Thanks, fixed.

Reply:
If fixes are so easy/straight forward then just go ahead and resolve the convos. 👍